wiki/Ranking_Criteria/osu!taiko/ko.md

```diff
@@ -1,61 +1,163 @@
-osu!taiko Ranking Criteria
-====================================
+osu!taiko 
+=============
+
+_Main page: [Ranking Criteria](/wiki/Ranking_Criteria)_
+
+이곳에 적힌 룰 및 가이드라인은 [Ranking Criteria subforum](https://osu.ppy.sh/community/forums/87)에서 논의된 내용입니다. 포럼에서 새로운 내용을 제안할 수 있으며, 합의가 된다면 이곳에 내용이 추가됩니다.
+
+용어
+--------------
+
+### 난이도 이름
+- ![Kantan](/wiki/shared/diff/easy-t.png "Kantan") **Kantan:** Easy
+- ![Futsuu](/wiki/shared/diff/normal-t.png "Futsuu") **Futsuu:** Normal
+- ![Muzukashii](/wiki/shared/diff/hard-t.png "Muzukashii") **Muzukashii:** Hard
+- ![Oni](/wiki/shared/diff/insane-t.png "Oni") **Oni:** Insane
+- ![Inner/Ura Oni](/wiki/shared/diff/expert-t.png "Inner/Ura Oni") **Inner/Ura Oni:** Expert
+
+대체 가능한 난이도 이름은 [이 스레드](https://osu.ppy.sh/community/forums/topics/178700)에서 확인할 수 있습니다.
+
+앞에 붙는 `Inner` 혹은 `Ura`는 기반이 되는 난이도에 비해 높은 난이도임을 나타내지만, 그것보다 높은 난이도에 비해서는 낮다는 것을 의미합니다 (예를 들자면, Ura Kantan은 Kantan 난이도보다는 어렵지만, Futsuu 난이도보다는 쉽습니다).
+
+### 게임 플레이 요소
+- **동 / 빨간 노트:** 드럼 중앙 (Drum Center)에 설정된 키로 처리할 수 있는 비트 (기본키는 `X`, `C`).
+- **캇 / 파란 노트:** 드럼 가장자리 (Drum Rim)에 설정된 키로 처리할 수 있는 비트 (기본키는 `Z`, `V`).
+- **큰 동 / 큰 빨간 노트 / 피니시 노트:** 드럼 중앙 (Drum Center)에 설정된 키를 동시에 치는 것으로 보너스 점수를 얻을 수 있는 강한 비트.
+- **큰 캇 / 큰 파란 노트 / 피니시 노트:** 드럼 가장자리 (Drum Rim)에 설정된 키를 동시에 치는 것으로 보너스 점수를 얻을 수 있는 강한 비트.
+- **BPM:** “beats per minute”의 약자로, 곡의 속도를 결정하는 요소.
+- **슬라이더 / 드럼 롤:** 틱이 있는 노란색 막대로, 설정된 키를 이용하여 플레이가 가능하다. 이 틱은 보통 곡의 1/4에 해당되는 지점에 스냅 되지만, BPM이 125 이하인 경우에는 1/8, 250 이상인 경우에는 1/2에 스냅 됩니다. 만약 틱 레이트가 3으로 설정되어 있으면, 1/4가 아닌 1/3에 스냅 됩니다.
+- **스피너:** 정해진 횟수만큼 동/캇을 번갈아가며 쳐야 하는 둥근 모양의 요소입니다. 횟수는 Overall Difficulty와 스피너의 길이에 따라 변화하며, 스피너 중앙에 표시됩니다.
+- **바 라인:** 마디의 시작을 표시하기 위해 플레이 필드에 나오는 눈금.
+- **오버랩:** 다른 노트를 부분적, 혹은 완전히 가리는 노트.
+- **휴식 구간:** 플레이어가 다음에 나올 패턴을 준비할 수 있도록 아무런 노트를 사용하지 않은 특정 구간.
+- **스트림:** 서클이 연속적으로 모인 부분. 일반적으로 ¼ 비트에 스냅 합니다.
+- **스냅:** 개체가 배치되는 타임 라인의 눈금.
+- **가변 스냅:** 노래의 변동하는 특성에 의해, 짧은 범위에 여러 방법으로 노트를 스냅하는 조합. 
+- **슬라이더 속도:** 플레이 필드 내의 노트/드럼 롤/스피너/눈금이 오른쪽에서 왼쪽으로 수평 이동하는 속도. 기본 슬라이더 속도는 타이밍 패널에서 제어가 가능하며, 상속된 포인트 (녹색 선)을 이용해 추가적으로 변경이 가능합니다.
+- **부드러운 변속:** 낮은 속도에서 높은 속도로, 혹은 높은 속도에서 낮은 속도로 점진적으로 전환될 수 있도록 처리하는 방법.
+- **즉흥 매핑:** 본래 가지고 있는 소리보다 더 많은 노트를 두는 것.
+- **태고 템플릿 배경:** 본래 태고의 달인과 비슷한 느낌이 되도록 하는 배경 이미지. 보통 검은색 라인과 함께 흰색 글씨로 아티스트, 타이틀을 표기합니다.
+
+## 일반
+
+### 규칙
+
+모든 **규칙**은 가이드라인이 **아니므로**, **어떠한 상황**에서도 **어기면 안됩니다**.
+
+- **각 노트는 이전 및 이후에 올 노트와 명확히 구분이 가능한 수 있는 색상을 가지고 있어야 합니다.**
+- **모든 음은 그것이 표현하려고 하는 음악 레이어나 노래가 제공하는 레이어를 강화하기 위한 레이어 유닛, 아니면 매퍼에 의해 즉흥적으로 만들어진 추가적인 레이어 유닛인지 분명하게 지정할 수 있어야 합니다.** 없는 속도 변화를 만들거나, 일반적인 음악의 움직임과 모순되거나, 노래의 강도를 잘못 해석하는 방식으로 즉흥 매핑을 하지 마세요. 즉흥 매핑은 노래의 레이어를 강화하거나, 새로운 레이어를 추가해야 합니다. 그렇지 않으면, 노래와의 관계를 포기하는 것이 되므로 리듬 게임의 주된 목적과는 모순되게 됩니다.
```

Comment:
There was an opinion that It's hard to understand the meaning of overall sentence.